Cargo.toml

```diff
@@ -10,13 +10,21 @@ repository = "https://github.com/stephaneyfx/brightness.git"
 documentation = "https://docs.rs/brightness"
 keywords = ["brightness", "backlight"]
 
+[features]
+async = ["async-trait", "futures", "tokio", "tokio-stream"]
+default = ["async"]
+
 [dependencies]
-async-trait = "0.1.50"
-futures = "0.3.16"
+async-trait = { version = "0.1.50", optional = true }
+cfg-if = "1.0.0"
+futures = { version = "0.3.16", optional = true }
+itertools = "0.10.3"
 thiserror = "1.0.26"
+tokio = { version = "1.20.1", features = ["rt", "macros", "fs"], optional = true }
+tokio-stream = { version = "0.1.9", features = ["fs"], optional = true }
 
 [target.'cfg(target_os = "linux")'.dependencies]
-zbus = "1.9.1"
+zbus = { version = "2", default-features = false, features = ["tokio"] }
```

Comment:
See: https://dbus.pages.freedesktop.org/zbus/faq.html?highlight=tokio#why-do-async-tokio-api-calls-from-interface-methods-not-work

Background: We need to use Tokio in order to do non-blocking file operations (Linux) and non-blocking Win32 calls on background threads (Windows). Upgrading to zbus 2 with these features ensures a compatible runtime with tokio.

Reply:
`tokio` is not strictly needed to handle blocking operations. As stated in the linked FAQ item, `tokio` is only needed when one has async functions using `tokio` and these functions are called by `zbus`.